pipeline/src/lib.rs

```diff
@@ -102,6 +105,7 @@
 mod dyn_nf;
 #[allow(clippy::module_inception)]
```

Comment:
Is this still required since we moved the crate?  It might be, not sure.

Reply:
I don't think it's required. Removing...